MediaPicker/build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'com.android.library'
     id 'org.jetbrains.kotlin.android'
+    id 'org.jetbrains.kotlin.plugin.parcelize' version '1.5.30-M1'
```

Comment:
🔧  By using Gradle's Plugin DSL we now should rather set up versions of our plugins in `settings.gradle`. I've pushed a commit on separate branch: 7409305d5cd8f2bde0b3345c35b062f4c6233c6a - if you'd like to add this change to the PR, please cherry-pick the commit to this PR.

This is similar, modern setup [we have in FluxC](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/develop/settings.gradle).

Reply:
TIL! Thank you :) I fixed this in 8ce600f following your advise